tests/cli/test_compose_sanity.sh

```diff
@@ -69,28 +69,29 @@ rlJournalStart
         fi
     rlPhaseEnd
 
+    rlPhaseStartTest "compose info"
```

Comment:
This isn't really needed, the previous block makes sure it is RUNNING (or fails).

Reply:
Right. That was explicitly designed as a phase to cover the `compose info` command, maybe because we didn't use it before when checking for the status or just duplicate.  Should I remove it  then ?

Reply:
May as well remove it.

Reply:
Removed